public/index.html

```diff
@@ -15,7 +15,7 @@
     -->
     <link rel="shortcut icon" href="%PUBLIC_URL%/tla-logo.png">
     <link href="https://fonts.googleapis.com/css?family=Chivo:300,400|Poppins:400,600,700" rel="stylesheet" />
-    <link href="https://fonts.googleapis.com/css?family=Anonymous+Pro|Ubuntu+Mono" rel="stylesheet">
```

Comment:
Was Anonymous Pro being used for anything?

Reply:
Doesn't look like it; i'll merge

Reply:
Yes. Anonymous Pro wasn't used anywhere.